### DISCUSSION TITLE
guide/init

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#17] [Announcements]
# guide/init

include(./init.snippet.md) 初始化指引 初始化流程包含四部分： 配置用户: 必填; 基本配置: 网站运行的必要配置; 高级配置: 可选，用于开启一些高级功能; 布局配置: 可选，配置前台布局;

https://vanblog.mereith.com/guide/init.html

**Comment by BlackFlyingFish** [comment]
卡在这一步没有反应，能讲解一下如何设置域名吗？

**Reply by Mereithhh** [reply]
具体是怎么没有反应呢

**Comment by Paimon-114514** [comment]
卡在这一步没有反应，能讲解一下怎么用IP访问并初始化吗？

**Comment by jiechliu** [comment]
牛逼
